01_install_requirements.sh

```diff
@@ -21,13 +21,13 @@ if ! which docker-machine-driver-kvm2 2>/dev/null ; then
 fi
 
 if ! which kubectl 2>/dev/null ; then
-    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
Can you split the unrelated quoting changes out into another patch?  That would make it easier to review the most important changes in this PR.